packages/components/src/border-box-control/styles.ts

```diff
@@ -16,16 +15,23 @@ export const BorderBoxControl = css``;
 
 export const LinkedBorderControl = css`
 	flex: 1;
+	${ rtl( { marginRight: '24px' }, { marginLeft: '24px' } )() }
+`;
+
+export const Wrapper = css`
+	position: relative;
 `;
 
 export const BorderBoxControlLinkedButton = (
-	__next36pxDefaultSize?: boolean
+	__next40pxDefaultSize?: boolean
 ) => {
 	return css`
+		position: absolute;
+		top: ${ __next40pxDefaultSize ? '8px' : '3px' };
+		right: 0;
```

Comment:
This used to be styled leveraging the flex box until a [revised design](https://github.com/WordPress/gutenberg/pull/41860#issuecomment-1257815773) required it to be positioned absolutely so that the split border controls for individual sides could stretch all the way to the right of the container.

I'll clean up the flex styles if they aren't required. If you have a better approach to achieving the design's layout, I'd be happy to implement it.

Reply:
I see, thank you for the context — absolute positioning is definitely the way to fo when the sides are unlinked.

When the sides are linked, you could go for flexbox but I see the convenience for keeping the absolute positioning.